src/create_particles.cpp

```diff
@@ -379,6 +379,7 @@ void CreateParticles::create_single()
   double temp_thermal = particle->mixture[imix]->temp_thermal;
   double temp_rot = particle->mixture[imix]->temp_rot;
   double temp_vib = particle->mixture[imix]->temp_vib;
+  double temp_elec = particle->mixture[imix]->temp_elec;
```

Comment:
Unless I missed something, none of these new variables are used in the code (only defined)? Please remove or update. Similar for other files: `fix_emit_face.cpp`, `fix_emit_face_file.cpp`, and `fix_emit_surf.cpp`

Reply:
This is almost certainly leftover from when the electronic energy was a new member of the particle class rather than using the custom data attribute. `fix_elecmode` has an update_custom method where we should set it, but apparently never propagated the `temp_elec` through. Is changing the signature on `fix::update_custom` to include `temp_elec` too invasive?

Reply:
>Is changing the signature on fix::update_custom to include temp_elec too invasive?

No I think it is reasonable, since it matches the other temps